olmo/eval/downstream.py

```diff
@@ -1863,3 +1863,503 @@ def doc_to_label(self, doc) -> int:
         {"dataset_path": "winogrande", "dataset_name": "rc_5shot", "metric_type": "bpb"},
     ),
 }
+
+# This standardizes the metrics we should eval for the ladder.
+# Train and test sets are added when applicable.
+# No subsampling happens in these sets.
+label_to_task_map_new = {
+    "arc_challenge_train_rc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "train_rc_5shot", "metric_type": "len_norm"},
+    ),
+    "arc_challenge_train_rc_5shot_bpb": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "train_rc_5shot", "metric_type": "bpb"},
+    ),
+    "arc_challenge_train_mc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "train_mc_5shot", "metric_type": "acc"},
+    ),
+    "arc_challenge_train_mc_5shot_bpb": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "train_mc_5shot", "metric_type": "bpb"},
+    ),
+    "arc_challenge_val_rc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "val_rc_5shot", "metric_type": "len_norm"},
+    ),
+    "arc_challenge_val_rc_5shot_bpb": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "val_rc_5shot", "metric_type": "bpb"},
+    ),
+    "arc_challenge_val_mc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "val_mc_5shot", "metric_type": "acc"},
+    ),
+    "arc_challenge_val_mc_5shot_bpb": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "val_mc_5shot", "metric_type": "bpb"},
+    ),
+    "arc_challenge_test_rc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "test_rc_5shot", "metric_type": "len_norm"},
+    ),
+    "arc_challenge_test_rc_5shot_bpb": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "test_rc_5shot", "metric_type": "bpb"},
+    ),
+    "arc_challenge_test_mc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "test_mc_5shot", "metric_type": "acc"},
+    ),
+    "arc_challenge_test_mc_5shot_bpb": (
+        OEEvalTask,
+        {"dataset_path": "arc_challenge", "dataset_name": "test_mc_5shot", "metric_type": "bpb"},
+    ),
+    "arc_easy_train_rc_5shot": (
+        OEEvalTask,
+        {"dataset_path": "arc_easy", "dataset_name": "train_rc_5shot", "metric_type": "len_norm"},
+    ),  # this used to be acc
```

Comment:
Can we safely change acc to len_norm here? I want to match what arc_challenge is doing.